jenkins/branch/autest.pipeline

```diff
@@ -79,14 +97,35 @@ pipeline {
 			steps {
 				echo 'Starting AuTest'
 				dir('src/tests') {
-					sh '''
-						set +e
+
+					sh '''#!/bin/bash -x
```

Comment:
I think @ezelkow1 observed that these caused issues with the FreeBSD system. It looked like, judging from poking around online, that (1) there's no good universal solution, and (2) the optimal strategy was to use `#!/usr/bin/env bash`.